package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "build": "webpack -p --config webpack.production.config.js",
     "start": "babel-node server/server.js",
-    "lint": "eslint --ignore-path .gitignore .",
+    "start-production": "NODE_ENV=production node dist/server.js",
```

Comment:
Lets hold the `foreverjs` until we gain some experience with Heroku and the dynos. Ideally, we should use static site hosting, and not an express backend.